server/repository/src/migrations/v2_06_00/add_report_sync.rs

```diff
@@ -0,0 +1,22 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "report"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        if cfg!(feature = "postgres") {
+            // Postgres changelog variant
+            sql!(
+                connection,
+                r#"
+                    ALTER TYPE changelog_table_name ADD VALUE IF NOT EXISTS 'om_report';
```

Comment:
Table name as 'om_report' to distinguish between omSupply central sync records, and historic legacy syncing of reports.

I've kept the name elsewhere as 'report' for consistency with naming elsewhere in the OMS repo

Reply:
Should be 'report' i think no to fail postgres tests